test/openssl/test_ssl.rb

```diff
@@ -838,7 +840,9 @@ def test_post_connection_check_wildcard_san
 
     # LibreSSL 3.5.0+ doesn't support other wildcard certificates
     # (it isn't required to, as RFC states MAY, not MUST)
-    return if libressl?
+    if libressl?
+      omit "LibreSSL 3.5.0+ doesn't support some wildcard certificates"
+    end
```

Comment:
https://github.com/ruby/openssl/pull/861 is an alternative to this. This test case should run with LibreSSL too.